Neo4j.Driver/Neo4j.Driver.Tests.TestBackend/Protocol/DriverQuery/ExecuteQuery.cs

```diff
@@ -132,5 +139,6 @@ public class ExecuteQueryConfigDto
         public int? timeout { get; set; }
         [JsonConverter(typeof(QueryParameterConverter))]
         public Dictionary<string, CypherToNativeObject> txMeta { get; set; }
+        public AuthorizationToken auth { get; set; }
```

Comment:
this has been renamed to authorizationToken

Reply:
Done